### PR TITLE
refactor: remove reset

### LINKED ISSUE #27542: remove .reset-component
### Background

目前多个组件样式引入 .reset-component，造成样式重写了多份。

- https://github.com/ant-design/ant-design/issues/9363#issuecomment-708375579

### Ref

- https://github.com/ant-design/ant-design/issues/9363#issuecomment-444905848
- https://github.com/ant-design/ant-design/issues/4331#issuecomment-423691027

### PR DESCRIPTION
**Pull request by xrkffgg**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [x] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
close https://github.com/ant-design/ant-design/issues/27542

### 💡 Background and solution
😂😂 先看看，不合，不确定有没问题。

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Comment by xrkffgg** [comment]
顺便看看 能减多少

**Comment by xrkffgg** [comment]
2 KB 啊，一笔巨款

**Comment by xrkffgg** [comment]
image

**Comment by xrkffgg** [comment]
粗略的找了一下，影响好大

**Comment by xrkffgg** [comment]
影响太多，方案暂时废弃。

File: components/style/core/base.less [hunk 1]
```diff
@@ -14,4 +16,15 @@
   *::after {
     box-sizing: border-box; // 1
   }
+
+  &,
+  * {
```
Comment (afc163) [inline components/style/core/base.less:21]: 不要加 *，给顶层元素就行了。

**Comment by afc163** [comment]
> 粗略的找了一下，影响好大

遇到啥问题？

File: components/style/core/base.less (hunk 1 above)
Comment (xrkffgg) [reply components/style/core/base.less:21]: 我先去掉看看

**Comment by yoyo837** [comment]
不仅仅是*, 还有global的 标签选择器也需要去掉：`div`、`span`什么的.

**Comment by afc163** [comment]
确实不行，会影响内部元素，要搞个组件白名单，手动或自动化搞成下面的代码：

```jsx
.ant-menu, .ant-btn, .ant-xxx, .ant-yyy, ..., .ant-zzz {
  .reset;
}
```

**Comment by hengkx** [comment]
吧argo放进来 看看 预览效果

**Comment by zombieJ** [comment]
我倾向于保留它，看看能不能找个 webpack 插件把和默认相同的移除掉

**Pull request closed without merging**